README.md

```diff
@@ -56,15 +56,34 @@ $ nv .env mix <mix command>
 
 ## 🏎 Quickstart
 
+_This is the full developpement setup. To simply run the app, see the *Docker* instructions_
+
 1. If you don’t already have it, install `nodejs` with `brew install nodejs`
 2. If you don’t already have it, install `elixir` with `brew install elixir`
 3. If you don’t already have it, install `libyaml` with `brew install libyaml`
 4. If you don’t already have it, install `postgres` with `brew install postgres` or the [macOS app](https://postgresapp.com/)
```

Comment:
Is this really required if we go the docker-compose route?

Reply:
As the first sentence of the paragraph says, this is the full developpement setup. I think it’s important to list the dependencies in the README to not force the user on using Docker.

I will change the second sentence to say that the DB can also be started from Docker instead of just saying _To simply run the app_. 👍

Reply:
"developpement" is a typo